extensions/editor.js

```diff
@@ -3,3 +3,14 @@
  */
 import './shared/public-path';
 import './shared/block-category';
+import analytics from '../_inc/client/lib/analytics';
+
+// @TODO Please make a shared analytics solution and remove this!
```

Comment:
yes, this solution ( though working) is duplicating code. We have a selector for that data in the Redux state
https://github.com/Automattic/jetpack/blob/2fee69dd0a32f6e353cc4d47417d7ac8aa83b9c6/_inc/client/main.jsx#L298, so maybe along those lines, a wrapper to Jetpack_Editor_Initial_State would limit future code *-plications and errors.